rwslib/builders_example.py

```diff
@@ -228,7 +231,7 @@ def example_metadata(study_name, draft_name):
         projectname = 'Mediflex'
         odm_definition = example_clinical_data(projectname,"DEV")
         request = PostDataRequest(str(odm_definition))
-    print str(odm_definition)
+    print(str(odm_definition))
```

Comment:
Is this print for debugging and should it still be here?

Reply:
It predates me I think, will remove